.changeset/quick-needles-cross.md

```diff
@@ -0,0 +1,5 @@
+---
+"@nextui-org/popover": major
```

Comment:
@kght6123 

```suggestion
"@nextui-org/popover": patch
```

Update the changeset to be a `patch`, when creating it just press enter without selecting that it is a `major` / `minor`